src/main/java/Scruffles.java

```diff
@@ -0,0 +1,225 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Scruffles {
+    private static void printList(String filePath) throws FileNotFoundException {
+        File newFile = new File(filePath);
+        Scanner scan = new Scanner(newFile);
+        while (scan.hasNext()) {
+            System.out.println(scan.nextLine());
+        }
+    }
+
+    private static void loadArray(String filePath, ArrayList<Task> array) throws FileNotFoundException {
+        File newFile = new File(filePath);
+        Scanner scan = new Scanner(newFile);
+        while (scan.hasNext()) {
+            String taskString = scan.nextLine();
+            String[] inputArray = taskString.split(" / ");
+            Task task = new Task("");
+            boolean isDone = inputArray[1].equals("X");
+
+            switch (inputArray[0]) {
+                case "T":
+                    task = new Todo(inputArray[2], isDone);
+                    break;
+                case "D":
+                    task = new Deadline(inputArray[2], LocalDate.parse(inputArray[3]), isDone);
+                    break;
+                case "E":
+                    task = new Event(inputArray[2], LocalDate.parse(inputArray[3]), LocalTime.parse(inputArray[4]),
+                            LocalTime.parse(inputArray[5]), isDone);
+                    break;
+                default:
+                    continue;
```

Comment:
Perhaps the switch statement should have the following form: No indentation for case clauses?